csi/moac/test/node_test.js

```diff
@@ -12,7 +12,7 @@ const { MayastorServer } = require('./mayastor_mock');
 const enums = require('./grpc_enums');
 
 const UUID = 'ba5e39e9-0c0e-4973-8a3a-0dccada09cbb';
-const EGRESS_ENDPOINT = '127.0.0.1:12345';
```

Comment:
meh! thanks, for renaming this @jkryl of all bad names, this one is the worst :see_no_evil: